doc/user-guides/ratelimiting/multi-rlp-multi-listener.md

```diff
@@ -1,43 +1,19 @@
-# Gateway Rate Limiting for Cluster Operators
+# Rate limiting for ingress gateway
```

Comment:
This was just confusing i didnt understand what cluster operators mean. So it would be confusing for users to

Reply:
Same comment as https://github.com/Kuadrant/kuadrant-operator/pull/964/files#r1854464214

Reply:
Maybe a reference to Gateway API docs could help on the confusion